pkg/controller/template/constants.go

```diff
@@ -7,6 +7,9 @@ const (
 	// SetupEtcdEnvKey is the key that references the setup-etcd-environment image in the controller
 	SetupEtcdEnvKey string = "setupEtcdEnvKey"
 
+	// GCPRoutesController is the key that references the gcp-routes-controller image in the controller
+	GCPRoutesController string = "gcpRoutesController"
```

Comment:
change to "GCPRoutesControllerKey" to maintain consistent naming here.

Reply:
done with https://github.com/openshift/machine-config-operator/pull/1031/commits/a60e68569d9cdfda0ff0d00ca770bb740396437f

Reply:
thanks!